### PR TITLE
Set up a scheduled backup by default

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Set up a scheduled backup by default

**Related issue(s)**

See #4969

### CODE REVIEW

File: docs/backup/05-01-velero.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ This table lists the non-required configurable parameters, their descriptions, a
 
 | Parameter | Description | Default value |
 |-----------|-------------|---------------|
+| **schedules** | Sets up a scheduled backup | A scheduled backup runs at 07:00 on every day-of-week from Monday through Friday. |
```
Comment (clebs) [inline docs/backup/05-01-velero.md]: Maybe here we should also add the couple of values that can be set to change the scheduled backup, like `schedules.scheduled-backup.schedule` and `schedules.scheduled-backup.template.ttl`.

File: docs/backup/01-01-backup.md [hunk 2]
```diff
@@ -6,4 +6,6 @@ Kyma integrates with [Velero](https://github.com/heptio/velero/) to provide back
 
 Velero backs up Kubernetes resources and stores them in buckets of [supported cloud providers](https://velero.io/docs/v1.0.0/support-matrix/). It triggers physical volume snapshots and includes the snapshot references in the backup. Velero can create scheduled or on-demand backups, filter objects to include in the backup, and set time to live (TTL) for stored backups.
 
+If you provide the needed configuration for the Velero installation as explained [here](/components/backup/#installation-installation), Kyma by default sets up a scheduled backup that runs once a day every day from Monday to Friday. It takes a backup using the provided configurations. To change this behavior or its configurations, change **schedules** configuration. See more in this [guide](/components/backup/#configuration-velero-chart).
```
Comment (tomekpapiernik) [inline docs/backup/01-01-backup.md]:
```suggestion
If you configured Velero when installing Kyma as explained [here](/components/backup/#installation-installation), backup is enabled with the default schedule and runs once a day every day from Monday to Friday.  To change the settings of the backup change the **schedules** configuration in the Velero chart [configuration](/components/backup/#configuration-velero-chart).
```

File: docs/backup/05-01-velero.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (suleymanakbas91) [reply docs/backup/05-01-velero.md]: Done

File: docs/backup/08-01-create-backup.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ kubectl apply -f {filename}
     Scheduled backup
     </summary>
 
-To set up periodic backups, use Schedule custom resources. Deploy Schedule custom resources in the `kyma-system` Namespace to instruct the Velero Server to schedule a cluster backup. Make sure the indentation is correct.
+Kyma already comes with a periodic backup which is scheduled to run once a day every day from Monday to Friday. To set up other periodic backups, use Schedule custom resources. Deploy Schedule custom resources in the `kyma-system` Namespace to instruct the Velero Server to schedule a cluster backup. Make sure the indentation is correct.
```
[the remark this answers is not in the document]
Comment (suleymanakbas91) [reply docs/backup/08-01-create-backup.md]: I created this follow-up ticket for this: https://github.com/kyma-project/kyma/issues/5158

File: docs/backup/05-01-velero.md [hunk 4]
```diff
@@ -35,6 +35,7 @@ This table lists the non-required configurable parameters, their descriptions, a
 
 | Parameter | Description | Default value |
 |-----------|-------------|---------------|
+| **schedules** | Sets up a scheduled backup. By default, a scheduled backup runs at 07:00 on every day-of-week from Monday through Friday. | scheduled-backup |
```
Comment (tomekpapiernik) [inline docs/backup/05-01-velero.md]:
```suggestion
| **schedules** | Sets up a scheduled backup. By default, a scheduled backup runs at 07:00 daily on Monday through Friday. | scheduled-backup |
```

Comment (tomekpapiernik) [reply docs/backup/05-01-velero.md]: If scheduled-backup is the default value then you should format it as `code`.

File: docs/backup/08-01-create-backup.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bszwarc) [reply docs/backup/08-01-create-backup.md]: These are CRs based on Velero CRDs. As far as I know, we document Kyma CRDs separately.

**Pull request merged**